src/modules/UI/scenes/ChangeMiningFee/ChangeMiningFee.ui.js

```diff
@@ -36,52 +32,52 @@ export default class ChangeMiningFee extends Component<Props, State> {
   }
 
   componentWillUnmount () {
-    if (this.state.feeSetting !== this.props.feeSetting) {
-      this.props.onSubmit(this.state.feeSetting)
-    }
+    this.props.onSubmit(this.state.feeSetting)
   }
 
-  handlePress = (feeSetting: string) => this.setState({ feeSetting });
-  // handleChange = (fee: string) => this.setState({ fee: fee.replace(/\D/g, '') });
+  handlePress = (feeSetting: string, cb: any) => {
+    return this.setState({ feeSetting }, cb)
+  }
 
   render () {
     const { feeSetting } = this.state
 
     return (
-      <SafeAreaView>
-        <View style={style.container}>
-          <Gradient style={style.gradient} />
-          <View style={style.headerContainer}>
-            <Text style={style.header} >
-              {s.strings.change_mining_fee_body}
-            </Text>
-          </View>
-          <View style={style.body}>
-            {feeOptions.map(({ value, label }) => (
-              <View key={value} style={style.row}>
-                <RadioButton
-                  value={value}
-                  label={s.strings[label]}
-                  onPress={this.handlePress}
-                  isSelected={value === feeSetting}
-                />
-              </View>
-            ))}
-          </View>
-          {/* feeSetting === FEE.CUSTOM_FEE
-          && <View>
-            <TextInput
-              style={style.input}
-              value={fee}
-              keyboardType='numeric'
-              placeholder='Satoshi per byte'
-              onChangeText={this.handleChange}
-              returnKeyType='done'
-            />
+        <View style={styles.scene}>
```

Comment:
It looks like removing the `SafeAreaView` caused changes to the header area.
Was this intended?

Reply:
Yep, It had too much of a gap at the top and removing the SafeAreaView fixed it